### ISSUE TITLE
Search list items 

### ISSUE THREAD

**Issue by mongoux** [microsoft-search/pnp-modern-search#447]
Hi, 
not sure if this is the correct place for this question. 
Is it possible to search in list and display row information?

I have a list called current known problems and I would like the user to be able to search for terms in the error message column and for the search results to show that particular list row information. 

Thanks

**Comment by wobba** [comment]
That's how it works today when you are in the list and search. So yes it's possible. You need to scope down search to only the list items which can be achieved using a `path` property query or `SPContentType` property or similar.. depending on how your list is created. If you want to target the content in that column only you need to fine the columns corresponding crawled property and map it to a queryable managed property. You can get some information at https://docs.microsoft.com/en-us/sharepoint/technical-reference/crawled-and-managed-properties-overview, and I cover this as well a bit in my "Troubleshooting Search in SharePoint Online" book.

Does this help?

**Comment by mongoux** [comment]
I've managed to find the list column corresponding crawled property and mapped it to a queryable managed property and selected it in the search result Selected Properties selection. In the result layout details list I then added a new managed column and added a managed property: {{ErrorMessage01}} value, but the column content is still not displayed in the search results. 
![image](https://user-images.githubusercontent.com/35407310/91917338-e8640880-ecbf-11ea-9129-ea27bc70a1c2.png)
![image](https://user-images.githubusercontent.com/35407310/91917456-45f85500-ecc0-11ea-99fb-cdfc4f7fba9c.png)

any help would be appreciated. Thanks
